concurrency/generator/components/ceasar.go

```
@@ -0,0 +1,45 @@
+		}
+	}
+
+	return string(result)
+}
+
+// CaesarDecrypt reverses the Caesar encryption
+func CaesarDecrypt(text string, shift int) string {
+	return CaesarEncrypt(text, 26-shift)
+}
+
+func main() {
```

golangci-lint (1.64.8)

func `main` is unused

(unused)